packages/babel-compat-data/scripts/download-compat-table.sh

```diff
@@ -2,6 +2,12 @@
 set -e
 
 COMPAT_TABLE_COMMIT=4e9369a699b0e15ba5c21586ce3bdd34299db9c1
+GIT_HEAD=build/compat-table/.git/HEAD
+
+if [ -f "$GIT_HEAD" -a "$(cat $GIT_HEAD)" == $COMPAT_TABLE_COMMIT ]; then
```

Comment:
It feels ad-hoc to rely on git internals. How about
```suggestion
if [ -f "build/compat-table" -a "$(git rev-parse HEAD)" == $COMPAT_TABLE_COMMIT ]; then
```